config/passphrase.txt

```diff
@@ -0,0 +1 @@
+32guns
```

Comment:
This seems to be the passphrase previously needed for PEM and should not be included in the PR